rest/config_startup.go

```diff
@@ -102,13 +103,16 @@ type BootstrapConfig struct {
 }
 
 type APIConfig struct {
-	PublicInterface  string `json:"public_interface,omitempty"  help:"Network interface to bind public API to"`
-	AdminInterface   string `json:"admin_interface,omitempty"   help:"Network interface to bind admin API to"`
-	MetricsInterface string `json:"metrics_interface,omitempty" help:"Network interface to bind metrics API to"`
-	ProfileInterface string `json:"profile_interface,omitempty" help:"Network interface to bind profiling API to"`
+	PublicInterface     string `json:"public_interface,omitempty"  help:"Network interface to bind public API to"`
+	AdminInterface      string `json:"admin_interface,omitempty"   help:"Network interface to bind admin API to"`
+	MetricsInterface    string `json:"metrics_interface,omitempty" help:"Network interface to bind metrics API to"`
+	DiagnosticInterface string `json:"diagnostic_interface,omitempty" help:"Network interface to bind diagnostic API to"`
```

Comment:
This information should be added to https://github.com/couchbase/sync_gateway/blob/master/docs/api/components/schemas.yaml#L2014